Example/MiniApp/AppDelegate.swift

```diff
@@ -6,9 +6,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        
+
```

Comment:
What are all the whitespace-only changes for?

Reply:
By default swiftlint is throwing violation errors for these whitespaces, so I had fixed them. Do you want me to rollback these whitespace changes ?

Reply:
No you can keep them - if it's from swiftlint that's fine